modules/global/test/com/haulmont/bali/util/StringHelperTest.java

```diff
@@ -0,0 +1,37 @@
+package com.haulmont.bali.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Dmitry Chigileychik
+ * @since 08.02.17.
+ */
+public class StringHelperTest {
+
+    @Test
+    public void testEmptyStringNormalize() {
+        String result = StringHelper.removeExtraSpaces("");
```

Comment:
You said it throws StringOutOfBoundsException for empty string input. How does this test work then?